packages/demo-html/public/widget-html.html

```diff
@@ -12,7 +12,13 @@
     </style>
   </head>
   <body>
-    <div id="wrapper" data-tf-widget="moe6aa" data-tf-medium="unit-test"></div>
+    <div
+      id="wrapper"
+      data-tf-widget="moe6aa"
+      data-tf-medium="unit-test"
+      data-tf-transitive-url-parameters="foo,bar"
```

Comment:
Typo? Should be `data-tf-transitive-search-parameters`?

Reply:
yeah might be, maybe when I rebased